common/djangoapps/student/views.py

```diff
@@ -734,9 +735,16 @@ def dashboard(request):
         'nav_hidden': True,
         'course_programs': course_programs,
         'disable_courseware_js': True,
-        'xseries_credentials': xseries_credentials,
+        'xseries_credentials': xseries_credentials
```

Comment:
Why are we removing this comma?  I thought our convention was to end dicts with commas